### ISSUE TITLE
Cannot delete old token in iOS App

### ISSUE THREAD

**Issue by Jotec1705** [freeotp/freeotp-ios#370]
Hi, I want to delete several unused tokens in my iOS app. I haven’t found any way to do this. E.g. swiping left or right doesn’t do anything.

My Device:
iPhone 16 Pro
OS: 18.0.1

**Comment by vl4dt** [comment]
I just encounter the same problem, I would like to remove one I added twice by mistake, there is also no way to rename a token.

**Comment by JohnnyBoy123123** [comment]
Salme issue here. Incredible lack in functionality.

FreeOTP 2.3.5
IOS 18.3.1

**Comment by linwiz** [comment]
Swipe the entry to the right

**Comment by justin-stephenson** [comment]
Please confirm which iOS version and device does not allow 'swipe' to remove the token?

**Comment by JohnnyBoy123123** [comment]
> Swipe the entry to the right

Yes, thank you.

**Comment by linwiz** [comment]
> Please confirm which iOS version and device does not allow 'swipe' to remove the token?

The issue seems to be that the 'feature' to remove the item is unknown; people don't know the feature is there (swiping to remove), not that it doesn't work.

**Comment by Tomtiger66** [comment]
I have the same issue, I can't remove unsed tokens on IOS. 26.5 on Iphone 13, nor its posible to edit a token, for example to change the icon or ad unlock phone protection to include the token into backups.

**Comment by MichaelLoibl** [comment]
I have the same issue. Swiping does not work unfortunately. I use FreeOTP 2.3.6 on iPadOS 26.3

**Comment by MichaelLoibl** [comment]
This comment from another issue ([https://github.com/freeotp/freeotp-ios/issues/326]) helped me:
"Just in case someone comes across this issue, you can swipe the entry to the right. However the ergonomy of the gesture is a little touchy as a long press will trigger a change of relative position of the entry (and once in this mode you can't trigger the deletion). The gesture needs to be light and quick and a large move is not required, even a quick narrow right move will do the trick and trigger the appearance of the question : "Remove token".

Bottom line: Don't press long without moving, move your finger right quickly after you touch the screen on the entry you want to remove."

**Comment by Tomtiger66** [comment]
Hello Michael,Grade that’s worked for me too.I tried several times to remove a wrong entry in FreeOTP. I thought I’m an experience user of an iPhone, but I don’t reach to delete the wrong entry.this little tutorial cost that it worked with the first try.so thank you very much and have a nice day. 😀ThomasEl 06.08.2026, a la(s) 05:17, MichaelLoibl ***@***.***> escribió:﻿MichaelLoibl left a comment (freeotp/freeotp-ios#370)
This comment from another issue ([https://github.com//issues/326]) helped me:
"Just in case someone comes across this issue, you can swipe the entry to the right. However the ergonomy of the gesture is a little touchy as a long press will trigger a change of relative position of the entry (and once in this mode you can't trigger the deletion). The gesture needs to be light and quick and a large move is not required, even a quick narrow right move will do the trick and trigger the appearance of the question : "Remove token".
Bottom line: Don't press long without moving, move your finger right quickly after you touch the screen on the entry you want to remove."
